src/components/standalone/StandaloneAppLogin.vue

```diff
@@ -120,84 +151,183 @@ function validate() {
   }
   return isValidationOk
 }
+
+function validateOtp() {
+  clearErrors()
+  errorBag.value.clear()
+  let isValidationOk = true
+
+  const otpValidation = validateSixDigitCode(twoFaOtp.value)
+  if (!otpValidation.valid) {
+    errorBag.value.set('otp', [String(otpValidation.errMessage)])
+    if (isValidationOk) {
+      isValidationOk = false
+      focusElement(twoFaOtpRef)
+    }
+  }
+  return isValidationOk
+}
+
+async function verifyOtp() {
+  const isValidationOk = validateOtp()
+  if (!isValidationOk) {
+    return
+  }
+  loading.value.verifyOtp = true
+
+  try {
+    await verifyTwoFaOtp(username.value, jwtToken.value, twoFaOtp.value)
+    loginStore.loginSuccessful(username.value, jwtToken.value)
+  } catch (err: any) {
+    console.error(err)
+    if (err instanceof ValidationError) {
+      errorBag.value = err.errorBag
+    } else {
+      error.value.verifyOtp = t(getAxiosErrorMessage(err))
+      error.value.verifyOtpDetails = err.toString()
+    }
+  } finally {
+    loading.value.verifyOtp = false
+  }
+}
+
+function clearErrors() {
+  for (const [key, value] of Object.entries(error.value) as [string, any][]) {
+    if (typeof value === 'string') {
+      // @ts-ignore
+      error.value[key] = ''
+    } else if (Array.isArray(value)) {
+      // @ts-ignore
+      error.value[key] = []
+    }
+  }
+}
```

Comment:
Definitely not agreeing on error handling this way, it's error prone at best on the long run

Consider separating validation errors (the ones that end up below input fields) using the messagebag
And adding an Error object to wrap any unforeseeable error and display them differently

Reply:
This is just a quick way to empty `error` object, we do it like this in many other components. Refactoring error management for all components is out of the scope of this PR, but I'm gonna remove `clearErrors` function here

Reply:
True that this is not something that we do everywhere, but something that even remotely requires @ts-ignore is a real bummer, we lose even what the error we manage even is